src/api/java/dan200/computercraft/api/ComputerCraftAPI.java

```diff
@@ -117,7 +117,7 @@ public static IWritableMount createSaveDirMount( World world, String subPath, lo
 	 * @see dan200.computercraft.api.peripheral.IComputerAccess#mountWritable(String, IWritableMount)
 	 * @see dan200.computercraft.api.filesystem.IMount
 	 */
-	public static IMount createResourceMount( Class modClass, String domain, String subPath )
+	public static IMount createResourceMount( Class<?> modClass, String domain, String subPath )
```

Comment:
Please don't change things in another mod's api

Reply:
Oops, I didn't catch that this wasn't our API. I'll change this one back.

Reply:
I notice that this is already fixed upstream. Can we update our copy of their API?

Reply:
could, but not in this PR

Reply:
Ideally they should have a maven, or we add an api jar as a lib

Reply:
Okay, I'll make a separate PR for this.